src/cli/main.py

```
@@ -95,6 +98,18 @@ def main_callback(
     # before any command runs.
     install_on_root()
 
+    # F7 (hardening roadmap #159): sweep any interrupted durable_move
+    # operations from a prior crashed run. Runs before any command so
+    # the on-disk state is coherent before the user's intent executes.
+    # Failures here are logged + swallowed — a sweep error is never
+    # worth crashing the CLI over; the next run will retry.
+    try:
+        _durable_move_sweep(_default_journal_path())
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)